agents/redfish/fence_redfish.py

```diff
@@ -0,0 +1,152 @@
+#!@PYTHON@ -tt
+
+# Copyright (c) 2018 Dell Inc. or its subsidiaries. All Rights Reserved.
+
+# Fence agent for devices that support the Redfish API Specification.
+
+import sys
+import re
+import json
+import requests
+import atexit
+sys.path.append("@FENCEAGENTSLIBDIR@")
+
+from requests.packages.urllib3.exceptions import InsecureRequestWarning
+from fencing import *
+from fencing import fail_usage, run_delay
+
+def get_power_status(conn, options):
+    uri = options["--systems-uri"]
+    response = send_get_request(options, uri)
```

Comment:
uri variable is use just once and it will be readable with options["..."] as well. This is there multiple times but it has no real impact, so don't fix if you like it that way more.

Reply:
I will take a look. I prefer simpler code myself :)

Reply:
Only way I could simplify this is with:

`response = send_get_request(options, options["--systems-uri"])`

There is one instance when I call `send_get_request` where uri is not a part of options, so I need to pass a second parameter to specify the URI to use.

Reply:
If it's OK with you, left this as-is.

Reply:
My only suggestion here is that you could just use `options["--systems-uri"]` in place of uri, as uri is only used once the places I've seen it defined in the code.

Reply:
Done. Thank you @oalbrigt and @marxsk for your review. Let me know if you have other comments.